config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -904,7 +904,7 @@ periodics:
     path_alias: k8s.io/kubernetes
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/benchmarkjunit:latest
+    - image: gcr.io/k8s-staging-test-infra/benchmarkjunit:latest
```

Comment:
There is not tag for this image. Do we still want to do the flip ?

Reply:
Good catch, I'm looking into why the image is missing...

Reply:
Opened https://github.com/kubernetes/test-infra/pull/23771 to address

I'm going to use `gcrane` to manually sync something over in the meantime

Reply:
`gcrane copy gcr.io/{k8s-testimages,k8s-staging-test-infra}/benchmarkjunit:latest`